### PR TITLE
Fix tones after quantifiers in examples

### PR DESCRIPTION
**Pull request by lynn**

### CODE REVIEW

**Review by uakci — APPROVED** [review]

File: dictionary.json [hunk 1]
```diff
@@ -12771,7 +12771,7 @@
     "notes": [],
     "examples": [
       {
-        "toaq": "pèa sa dáqne",
+        "toaq": "pèa sa dảqnē",
```
Comment (robintown) [inline dictionary.json:12774]: :warning: T1 zay

File: dictionary.json [hunk 2]
```diff
@@ -12890,19 +12890,19 @@
     "notes": [],
     "examples": [
       {
-        "toaq": "rào sa ráı",
+        "toaq": "rào sa rảı",
         "english": "some time; at some point"
       },
       {
-        "toaq": "rào sıa dáqmoa",
+        "toaq": "rào sıa dảqmōa",
```
Comment (robintown) [inline dictionary.json:12897]: T1

File: dictionary.json [hunk 3]
```diff
@@ -12890,19 +12890,19 @@
...
         "english": "never, at no moment"
       },
       {
-        "toaq": "rào tu cháq",
+        "toaq": "rào tu chảq",
         "english": "every day"
       },
       {
-        "toaq": "rào tu dáqmoa",
+        "toaq": "rào tu dảqmōa",
```
Comment (robintown) [inline dictionary.json:12905]: T1

**Review by robintown — CHANGES_REQUESTED** [review]
The examples are outdated in lots of ways and could use a PR to more generally revise them, but I don't think there's a problem with fixing them incrementally.

**Review by robintown — APPROVED** [review]

**Pull request merged**